### PR TITLE
Get rid of static server

### PR DESCRIPTION
**Pull request by Morishiri**
As part of our proposal to secure static data I implemented FileController which exposes a route (requiring authentication) used to fetch image files.

### CODE REVIEW

File: yarn-error.log [hunk 1]
```diff
@@ -0,0 +1,3765 @@
+Arguments:
```
Comment (0x7f) [inline yarn-error.log:1]: @Morishiri should be removed from the commit.

File: yarn.lock [hunk 2]
```diff
@@ -0,0 +1,3719 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (0x7f) [inline yarn.lock:1]: Usually I use yarn as well. These days it is common to switch back to npm though. So you could instead add the `package-lock.json`. @Morishiri @ekorolev What do you think?

Comment (Morishiri) [reply yarn.lock:1]: I just used what was there already :) It's up to the authors which one to use, but I prefer npm as well ;)

**Comment by 0x7f** [comment]
@com2u this branch is now ready for merge + deploy (I don't have the permissions to merge). With this change, all images are hosted via the same endpoint and are now also protected by the logged in user as well. Before, the raw data was publicly accessible without login.

**Pull request merged**